maintainers/maintainer-list.nix

```diff
@@ -6825,6 +6825,12 @@
     githubId = 628342;
     name = "Tim Steinbach";
   };
+  nerdypepper = {
```

Comment:
Please put this into a separate commit with the message ``maintainers: add NAME`` and sort it alphabetically.